.build/website/src/pages/index.module.css

```diff
@@ -10,9 +10,8 @@
   overflow: hidden;
   background-repeat: no-repeat;
   background-position: top right;
-  /*sample for image*/
-  /*background-image: -webkit-image-set(url(https://everscale.network/images/Backgrounds/Main/main-hero.png) 1x, url(https://everscale.network/images/Backgrounds/Main/main-hero@2x.png) 2x);*/
-  /*background-image: image-set(url(https://everscale.network/images/Backgrounds/Main/main-hero.png) 1x, url(https://everscale.network/images/Backgrounds/Main/main-hero@2x.png) 2x);*/
+  background-image: -webkit-image-set(url(https://i.postimg.cc/y6pxJ9Px/166488327-602bbbae-6cbe-4e51-a445-a6d96b9436e2.jpg) 1x, url(https://i.postimg.cc/7w4mXzwd/166488327-602bbbae-6cbe-4e51-a445-a6d96b9436e1.jpg) 2x);
```

Comment:
`postimg` this is not an acceptable data source, use `arweave` or `docs.everscale.network` as a last resort `everscale.network`